### PR TITLE
initial documentation

### PR DESCRIPTION
**Pull request by mattHoyDev**

### CODE REVIEW

File: 01. DatatypesAndOperators.txt [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+3 // => 3
+
+// Math
+1 + 1 // => 2
+8 - 1 // => 7
+10 * 2 // => 20
+35 / 5 // => 7.0
+-5.0 / 3 // -> -1.67
+4 % 2 // => 0
+2 ** 3 // => 8
```
Comment (bengangllipson) [inline 01. DatatypesAndOperators.txt]: ^ instead of **?

File: 02. VariablesAndCollections.txt [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+int intVar1, intVar2: 123, 456
+
+// Float initialization
+float floatVariable: 1.11
+
+// Boolean initialization
+bool booleanVariable: True
+
+// List initialization. Lists are collections of any datatype
+list listVariable: [stringVariable, intVar1, booleanVariable
```
Comment (bengangllipson) [inline 02. VariablesAndCollections.txt]: missing closing bracket

File: 02. VariablesAndCollections.txt [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+// Array initialization. Arrays are collections of a single datatype
+arr arrayVariable: int[intVar1, intVar2]
+
+// Dictionary initialization. Dictionaries are a key, value store
+dict dictionaryVar: {
+	"key1": arrayVariable
+	"key2": 1
+	stringVariable: 
+}
```
Comment (bengangllipson) [inline 02. VariablesAndCollections.txt]: This seems weird, did you mean

    dict dictionaryVar: {
    	"key1": arrayVariable,
    	"key2": stringVariable
    }

File: examples.txt [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+str stringVariable: "Hello"
+int intVar1, intVar2: 123, 456
+bool booleanVariable: True
+list listVariable: [stringVariable, intVar1, booleanVariable]
+arr arrayVariable: int[intVar1, intVar2]
+dict dictionaryVar: {
+	"key1": arrayVariable
+	"key2": 1
+	stringVariable: 
+}
```
Comment (bengangllipson) [inline examples.txt]: same weird dict thing

File: examples.txt [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+float floatVariable: 1.11
+
+//Keywords
+str, int, bool,
+list, arr, dict,
+float, range, len,
+if, else, for, 
+in, return, import,
+assert, func
```
Comment (bengangllipson) [inline examples.txt]: "ind" or "with" for index

File: examples.txt [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+
+//flow
+if ( intVar1 = 1 ) (
+	print("intVar1 = 1")
+) else if ( intVar1 = 123 ) (
+	print("intVar1 = 123")
+) else (
+	print("I don't know what intVar1 is.")
+)
```
Comment (bengangllipson) [inline examples.txt]: I kind of prefer having no spaces inside the if statments

    if (intVar1 = 1) (
        // do stuff
    )

same goes for for

    for (int i in len(arrayVariable)) (
        // do stuff
    )

Comment (mattHoyDev) [reply examples.txt]: Sure sure

File: examples.txt (hunk 5 above)
Comment (mattHoyDev) [reply examples.txt]: adding

File: 01. DatatypesAndOperators.txt (hunk 1 above)
Comment (mattHoyDev) [reply 01. DatatypesAndOperators.txt]: Hmmm. I must not have clicked save...

File: 02. VariablesAndCollections.txt (hunk 2 above)
Comment (mattHoyDev) [reply 02. VariablesAndCollections.txt]: ty

File: 02. VariablesAndCollections.txt (hunk 3 above)
Comment (mattHoyDev) [reply 02. VariablesAndCollections.txt]: I meant to highlight that we could use string vars as keys. Forgot to add a val

**Review by bengangllipson — APPROVED** [review]

**Pull request merged**